src/api/resources/sampleunitmethods/beltinvertmethod.py

```
@@ -0,0 +1,36 @@
+from rest_framework import serializers
+
+from ...models import BeltInvert, ObsBeltInvert
+from ..base import BaseAPISerializer
+
+__all__ = ["BeltInvertSerializer", "ObsBeltInvertSerializer"]
+
+
+class BeltInvertSerializer(BaseAPISerializer):
+    class Meta:
+        model = BeltInvert
+        exclude = []
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)